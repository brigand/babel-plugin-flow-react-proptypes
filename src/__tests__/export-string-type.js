const babel = require('babel-core');
const content = `
export type T = string;
export type TOptional = ?string;
`;

it('export-intersection-type', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['env', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')],
  }).code;
  expect(res).toMatchSnapshot();
});
