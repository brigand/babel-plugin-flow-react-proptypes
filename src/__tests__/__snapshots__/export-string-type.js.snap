// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`export-intersection-type 1`] = `
"\\"use strict\\";

var babelPluginFlowReactPropTypes_proptype_T = require(\\"prop-types\\").string;

if (typeof exports !== \\"undefined\\") Object.defineProperty(exports, \\"babelPluginFlowReactPropTypes_proptype_T\\", {
  value: babelPluginFlowReactPropTypes_proptype_T,
  configurable: true
});

var babelPluginFlowReactPropTypes_proptype_TOptional = require(\\"prop-types\\").string;

if (typeof exports !== \\"undefined\\") Object.defineProperty(exports, \\"babelPluginFlowReactPropTypes_proptype_TOptional\\", {
  value: babelPluginFlowReactPropTypes_proptype_TOptional,
  configurable: true
});"
`;
